bin/behat-prepare.sh

```diff
@@ -24,6 +24,13 @@ fi
 
 set -ex
 
+###
+# Apply any outstanding upstream updates.
+# This never happens manually, so we might as well do it in automation before we run tests.
+###
+terminus connection:set $TERMINUS_SITE.dev git
+terminus upstream:updates:apply $TERMINUS_SITE.dev --accept-upstream
```

Comment:
Improvement would be to check for upstream updates and only run these steps if we have to, but optional